apps/docs/app/root/layout/SiteSettings.tsx

```diff
@@ -24,13 +24,19 @@ export const SiteSettings = ({ showLabel }: SiteSettingsProps) => {
     <CardSelect
       variant="ghost"
       size="md"
-      className="dark"
       icon={<SettingsX1Fill24Icon />}
       withChevron={false}
       fontWeight="bold"
       {...labelProps}
+      position={"relative"}
+      className="dark"
     >
-      <Flex gap={4} flexDirection="column" maxWidth="30ch">
+      <Flex
+        gap={4}
+        flexDirection="column"
+        maxWidth="30ch"
+        width={["100%", "30ch"]}
+      >
```

Comment:
Kanskje det er out of scope for dette branch men det ser ut at dette elementet trenger en høyere z-index og en backgroundColor? zIndex="modal"